syntax = "proto3";

import "messages/basic.proto";

message ShowFiles {
    oneof optional_options {
        string options = 1;
    } 
}

message FileList {
    repeated File files = 1;
}
